README.md

```diff
@@ -6,7 +6,19 @@ Module 4 | Contributors: Autumn Martin
 Deciding which ski resort to go to is hard with factors to consider like epic snowfall, and if the wind and chill will be worth it. This app is for all skiers, snowboarders, and powder chasers. Ski meow helps you choose the ski resort with the most ideal conditions for a great powder day! Let's Ski Meow!
 
 #### Tech Stack
-HTML, CSS, JavaScript, Waffle
+HTML, CSS, JavaScript, Waffle, Webpack
 
 ## Getting started
-Clone this repository via `git clone git@github.com:Autumn-Martin/ski_meow.git`
+
+### Production
+This app has been deployed to GitHub Pages. To view the app in production, visit: https://autumn-martin.github.io/ski_meow/
+
+### Development
+First, clone this repository via `git clone git@github.com:Autumn-Martin/ski_meow.git`.
+
+Install dependencies via `npm install`.
+
+Fire up the development server with `npm start`. Wait until it is running.
+
+Now visit `http://localhost:8080/` in your browser to view the app
```

Comment:
Add link to deployed app in body and at top of GitHub repo next to description.

Reply:
Done!